aries-site/src/layouts/main/Footer.js

```diff
@@ -1,7 +1,12 @@
 import React, { useContext } from 'react';
 import Link from 'next/link';
-import { FooterLink } from 'aries-core';
```

Comment:
Is `<FooterLink />` used anywhere any more? If not, should the component be removed from aries-core?

Reply:
I think it should be removed. I didn't do it in this PR because I didn't want to add clutter, but I do think if the component in footer will ultimately just be a button, we should remove.